woocommerce/woocommerce-ios
# [Mobile Payments] UI for Enable Pay in Person toggle switch in IPP settings

Closes: #7591 
Closes: #7570



### Description


In 10.1, we added the ability for a merchant to turn on the `Cash on Delivery` payment gateway in the In-Person Payments Onboarding flow. There is currently no way to undo that choice in the app, nor to turn it on at a later date if the merchant chose to skip the initial onboarding prompt.

For iteration 2 of the project, we'll add a toggle to the In-Person Payments settings so that a user can see the current state of the gateway, and change it outside of the onboarding flow.

This PR tackles the UI of this new row:
- [x] Adds a new row to the IPP settings, including a title, descriptive text, help link* and toggle switch
- [x] Reflects the `enabled` state of the `cod` gateway on app launch, and after it's changed in the onboarding flow**.

*The help link is not tappable in this PR
**The switch itself can't be used to change the setting, and the PaymentGateways aren't refreshed except on app launch or change of site.

N.B. I've realised that we'll probably show this for every store, regardless of location or payment gateway status. I'll address that in a follow-up PR, we _probably_ don't want to show it to everyone even though it does have a valid use for non-IPP stores.

### Testing instructions


Using a US store with Cash on Delivery disabled (see wc-admin > Payments > All Payment Methods)

1. Go to the Menu tab, tap Payments
2. Observe that the new toggle shows as off
3. Tap `Continue Setup` and then `Enable Pay in Person`.
4. Observe when you go back to the menu, the toggle is on.

Turn Cash on Delivery back off using wc-admin.

1. Relaunch the app
2. Go to the Menu tab, tap Payments
3. Observe that the new toggle shows as off
4. Tap `Continue Setup` and then `Skip`.
5. Observe when you go back to the menu, the toggle is still off.

Turn Cash on Delivery back on using wc-admin.

1. Relaunch the app
2. Go to the Menu tab, tap Payments
3. Observe that the new toggle shows as on

### Screenshots



https://user-images.githubusercontent.com/2472348/187671499-a19cf3ee-fd35-409e-a167-a1f817dcd707.mp4


---
